### PR TITLE
feat(program-badge): added ebay-plus program badge

### LINKED ISSUE #1510: program-badge: add ebay-plus SVG
Add ebay plus SVG to skin

Screen Shot 2021-07-13 at 1 39 41 PM

### PR DESCRIPTION
**Pull request by austingardner**
## Description
Ebay Plus program badge.

## References
closes #1510 

## Screenshots
Screen Shot 2021-07-13 at 3 57 40 PM
Screen Shot 2021-07-13 at 3 57 47 PM

### CODE REVIEW

File: src/svg/ds6/program-badges.svg [hunk 1]
```diff
@@ -35,4 +35,14 @@
         <path fill="#3665f3" d="M21.29 16.88l2.08-4.46a.68.68 0 011.26 0l2.13 4.49a1 1 0 00.74.56l4.87.81a.8.8 0 01.38 1.35l-3.13 3.4a1 1 0 00-.26.76l.45 5.21v.08a.79.79 0 01-.29.78.63.63 0 01-.71.06L24.5 27.4a1 1 0 00-1 0l-4.32 2.51a.63.63 0 01-.71-.06.79.79 0 01-.29-.78.15.15 0 010-.07l.46-5.19a1 1 0 00-.26-.76l-3.14-3.4a.8.8 0 01.38-1.35l4.92-.84a1 1 0 00.75-.58z"/>
         <path fill="#3665f3" fill-rule="evenodd" d="M4 4.24v39.51a4.31 4.31 0 006.59 3.59L21.7 40.5a4.37 4.37 0 014.57 0l11.14 6.85a4.31 4.31 0 006.59-3.6V4.24A4.28 4.28 0 0039.69 0H8.31A4.28 4.28 0 004 4.24zm17.56 7.32a2.69 2.69 0 014.88 0l1.9 4 4.35.72A2.8 2.8 0 0134.21 21l-2.82 3.06.41 4.7a2.72 2.72 0 01-3.95 2.88L24 29.42l-3.83 2.22a2.72 2.72 0 01-3.95-2.88l.42-4.7L13.79 21a2.81 2.81 0 011.51-4.7l4.4-.74z"/>
     </symbol>
+    <symbol id="program-badge-ebay-plus" viewBox="0 0 58 16">
+        <style>
+          .st0{fill:#23804f}
```
Comment (ianmcburnie) [inline src/svg/ds6/program-badges.svg:40]: We should avoid inline styles. This should be a fill on the SVG instead. Or use a class from skin css.

Comment (austingardner) [reply src/svg/ds6/program-badges.svg]: Let me know if these recent changes address what you were thinking here.

**Review by ianmcburnie — APPROVED** [review]

File: src/svg/ds6/program-badges.svg (hunk 1 above)
Comment (ianmcburnie) [reply src/svg/ds6/program-badges.svg]: Looks good. I'll take care of merging master into 13.0.0 once the release goes out (there are going to be some ugly conflicts due to the changes I've made in 13.0.0 - prob best if I deal with it).

**Review by ianmcburnie — APPROVED** [review]
Looks good. I'll take care of merging master into 13.0.0 once the release goes out (there are going to be some ugly conflicts due to the changes I've made in 13.0.0 - prob best if I deal with it).

**Pull request merged**